service.subtitles.divxplanet/service.py

```diff
@@ -0,0 +1,351 @@
+# -*- coding: utf-8 -*-
+# encoding=utf8
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+from xbmc import log
+
+import os
+import re
+import shutil
+import string
+import urllib
+import requests
+from random import randint
+from time import sleep
+from BeautifulSoup import BeautifulSoup
+import sys
+reload(sys)  # Reload does the trick!
+sys.setdefaultencoding('UTF8')
+
+__addon__ = xbmcaddon.Addon()
+__author__ = __addon__.getAddonInfo('author')
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
```

Comment:
Doesnt need to be done for this pull request, but in the future  please get rid of `__varname__` naming style.